src/pages/Flow/Run.vue

```diff
@@ -763,6 +831,37 @@ export default {
 </style>
 
 <style lang="scss">
+.ma-2.debuglevel {
```

Comment:
in general we don't want to attach styles to built-in Vuetify class names, especially without specifying that the SCSS is scoped to the file (to do so, add "scoped" to line 833, but without digging I don't know if there's a reason we didn't do that on this file). Does this CSS work if you just specify `debuglevel` for the classname?

Also, did you check that all these color combos work in both light and dark modes?

Reply:
Also, our linter has a strong preference (if not fully erroring) for kebab-case in classnames

Reply:
This CSS needs `ma-2` to work unfortunatley, I'll move this into the scoped style section of the file

Reply:
Changed the font color to look a bit better in dark mode, good catch!